debian/copyright

```diff
@@ -1,59 +1,1130 @@
+  2011, 2017, MariaDB Corporation.
+  2011, Monty Program Ab
+  2011, Oleksandr Byelkin
+  Richard A. O'Keefe
+License: MIT
+
+Files: BUILD/FINISH.sh
+ BUILD/SETUP.sh
+ BUILD/autorun.sh
+ BUILD/cmake_configure.sh
+ scripts/mysql_fix_extensions.sh
+ scripts/mysql_setpermission.sh
+ scripts/mysqlaccess.sh
+ scripts/mysqldumpslow.sh
+ scripts/mysqlhotcopy.sh
+Copyright: 2000, 2010, Oracle and/or its affiliates.
+  2000, 2011, Oracle and/or its affiliates.
+  2000, 2017, Oracle and/or its affiliates.
+  2001, 2017, Oracle and/or its affiliates.
+  2005, 2010, Oracle and/or its affiliates.
+  2009-2010, Oracle and/or its affiliates.
+License: LGPL-2.0
+
+Files: client/completion_hash.h
+ mysys/my_port.c
+ strings/ctype-bin.c
+ strings/ctype-eucjpms.c
+ strings/ctype-uca1400.h
+ strings/ctype-uca.h
+ strings/ctype-ucs2.c
+ strings/ctype-ujis.c
+ strings/ctype-unidata.c
+ strings/ctype-utf8.c
+ strings/dtoa.c
+Copyright: 2000-2002, 2006, MySQL AB
+  2000, 2013, Oracle and/or its affiliates.
+  2000, 2017, Oracle and/or its affiliates.
+  2002-2007, MySQL AB & tommy@valley.ne.jp
+  2002, 2014, Oracle and/or its affiliates.
+  2002, MySQL AB
+  2002, MySQL AB & tommy@valley.ne.jp
+  2003, 2013, Oracle and/or its affiliates
+  2004, 2013, Oracle and/or its affiliates.
+  2007, 2012, Oracle and/or its affiliates.
+  2009, 2014, SkySQL Ab.
+  2009, 2020, MariaDB
+  2009, 2020, MariaDB Corporation.
+  2010, 2017, MariaDB Corporation
+  2017, 2020, MariaDB Corporation.
+  2021, MariaDB
+License: LGPL-2.0
+
+Files: scripts/mysqld_multi.sh
+License: LGPL-2.0
+
+Files: include/maria.h
+ include/myisamchk.h
+ include/my_rnd.h
+ include/thr_timer.h
+ mysys/my_safehash.c
+ mysys/my_safehash.h
+ mysys/thr_timer.c
+ sql/log_slow.h
+Copyright:   2003-2007, MySQL AB
+  2003, MySQL AB
+  2006-2008, MySQL AB, 2008-2009 Sun Microsystems, Inc.
+  2006, MySQL AB
+  2009, 2017, MariaDB Corporation.
+  2009, 2019, MariaDB Corporation.
+  2012, Monty Program Ab
+  2013, Monty Program Ab
+  2013, Monty Program Ab
+  2014, Monty Program Ab
+  2017, MariaDB Corporation.
+License: GPL-2.0+
+
+Files: client/async_example.c
+ tests/nonblock-wrappers.h
+ tests/async_queries.c
+Copyright: 2011, Kristian Nielsen and Monty Program Ab.
+License: LGPL-2.1+
+
+Files: scripts/galera_new_cluster.sh
+License: LGPL-2.1+
+
+Files: scripts/mysqld_safe.sh
+ support-files/mysql-multi.server.sh
+ support-files/mysql.server.sh
+Copyright: Abandoned 1996 TCX DataKonsult AB & Monty Program KB & Detron HB
+License: public-domain
+
+Files: dbug/dbug.c
+ dbug/dbug_long.h
+Copyright: Abandoned, 1987, Fred Fish
+License: public-domain
+
+Files: sql/socketpair.c
+Copyright: 2007, 2010, Nathan C. Myers <ncm@cantrip.org>
+License: BSD-3-clause
+
+Files: libmysqld/libmysql.c
+ sql-common/conf_to_src.c
+ sql-common/errmsg.c
+Copyright: 2000, 2014, Oracle and/or its affiliates
+  2002, 2006, MySQL AB, 2009 Sun Microsystems, Inc.
+  2020, MariaDB Corporation.
+  2009, 2020, MariaDB Corporation
+License: GPL
```

Comment:
given GPL isn't a SPDX identifier can/should be go GPL-2.0 based on the Oracle source?

If that's decided the text isn't quantifying more.

Reply:
Fairly sure GPL-1.0 was never used.

Reply:
Ok I've update it to GPL-2.0 as I assume that it's not GPL 1.x neither.